example/fake-server.yml

```diff
@@ -118,12 +118,12 @@ Resources:
   FakeProviderEc2Instance:
     Type: "AWS::EC2::Instance"
     Properties:
-      ImageId: ami-aff65ad2
+      ImageId: ami-03e7dd4efa9b91eda
       InstanceType: t2.micro
       AvailabilityZone: !Ref AZone
       IamInstanceProfile:
         Ref: InstanceProfile
-      # KeyName: uncomment and give a value if access to the instance is needed
+      KeyName: mhuffnag
```

Comment:
Is this just for testing?

Reply:
I left it there so that, if we ever need to ssh into the instance, at least one of us has access.